### ISSUE TITLE
Время торговой сессии (завязка на VWAP)

### ISSUE THREAD

**Issue by teeth2** [SBProX/SBProX-Dev#1872]
Пожалуйста, добавьте инструментам индивидуальный выбор начала торговой сессии. Или вшейте эту функцию в индикатор VWAP. 
Сейчас нереально настроить нужное отображение VWAP (конфигурация которого зависит от времени начала торговой сессии). Пример прилагаю - розовой стрелкой обозначено место начала торговой сессии, чтобы конфигурация VWAP в обоих источниках совпадала.
Все эти танцы с бубнами и настройками VWAP идентично другим источникам приходится выполнять, поскольку многократно запрошенные здесь отклонения VWAP пока не реализованы. Очень хотелось бы их увидеть в SbPro - ценный и технически просто реализуемый функционал. Который поднимет ценность программы, поскольку в настоящее время этот индикатор приобретается только за деньги и есть не для всех платформ.

**Скриншоты**
![image](https://user-images.githubusercontent.com/78958646/107760466-d7252a80-6d3a-11eb-8f42-470a22d89f96.png)

**Comment by WebSurfer001** [comment]
Поставьте время на инструменте +1

**Comment by teeth2** [comment]
А вы не согласны с тем, что это неудобно - бегать по трём программам, и каждый раз три разных времени приводить к одному? В одном ПО отклонения VWAP, в другом торговля, третье - SbPro.
В настройках торгового ПО поменять начало торговой сессии для выбранного инструмента не составляет труда. Сделать это для индикатора VWAP в SbPro тоже элементарно - поменять одну переменную ("Начало торговой сессии") на указанную пользователем константу.

**Comment by WebSurfer001** [comment]
Согласен... Я вам подсказал как решить в данный момент эту проблему, как это сделано у меня. Если вы хотите подискутировать сбпро сделали форум.
